dash/testing/plugin.py

```diff
@@ -25,6 +25,25 @@ def pytest_addoption(parser):
         help="Name of the selenium driver to use",
     )
 
+    dash.addoption(
+        "--headless",
+        action="store",
+        default=False,
+        help="Run tests in headless mode",
+    )
```

Comment:
could we use `"store_true"` so you can simplify to `pytest --headless`, rather than `pytest --headless True`?

Reply:
yep, I like the shorter version